rdr_service/dao/questionnaire_response_dao.py

```diff
@@ -541,8 +540,8 @@ def _populate_codes_and_answers(cls, group, code_map, answers, link_id_to_questi
                                     answer_length = len(answer.valueString)
                                     max_length = QuestionnaireResponseAnswer.VALUE_STRING_MAXLEN
                                     if answer_length > max_length:
-                                        err_msg = "String value too long (len=%d); must be less than %d"
-                                        raise BadRequest(err_msg % (answer_length, max_length))
+                                        err_msg = "String value too long (len={:d}); must be less than {:d}"
```

Comment:
shouldnt this be `answer_length` and `max_length` ?

Reply:
Those variables are being formatted in the next line, but the code could be a little cleaner. I'll move the formatted variables into the `err_msg` string.